pkg/services/sqlstore/migrations/alert_mig.go

```diff
@@ -82,4 +82,18 @@ func addAlertMigrations(mg *Migrator) {
 		{Name: "type", Type: DB_NVarchar, Length: 255, Nullable: false},
 		{Name: "settings", Type: DB_Text, Nullable: false},
 	}))
+	active_node := Table{
+		Name: "active_node",
+		Columns: []*Column{
+			{Name: "id", Type: DB_BigInt, IsPrimaryKey: true, IsAutoIncrement: true},
+			{Name: "node_id", Type: DB_NVarchar, Length: 255, Nullable: false},
+			{Name: "heartbeat", Type: DB_TimeStampz, Nullable: false},
+			{Name: "sequence", Type: DB_Int, Nullable: false},
+		},
+		Indices: []*Index{
+			{Cols: []string{"node_id", "heartbeat"}, Type: UniqueIndex},
```

Comment:
@sanchitraizada The index contains node_id and heartbeat. But are we going to the exact heartbeat for lookup? Are we supposed to normalise the heartbeat to nearest next minute? I think we should set the default heartbeat to CURRENT_TIMESTAMP (i.e. timestamp of db) and not set it on the node. This is to avoid issues when VMs time goes out of sync. Thoughts?